helix-core/src/main/java/org/apache/helix/util/InstanceValidationUtil.java

```diff
@@ -120,17 +121,17 @@ public static String getInstanceHelixDisabledType(HelixDataAccessor dataAccessor
    * @param clusterConfig
    * @return
    */
-  public static boolean isInstanceEnabled(InstanceConfig instanceConfig, ClusterConfig clusterConfig) {
+  public static boolean isInstanceEnabled(InstanceConfig instanceConfig, @Nullable ClusterConfig clusterConfig) {
     if (instanceConfig == null) {
```

Comment:
I feel like marking nullable for clusterConfig is a bit conflict with `isEnabled()` line 78. As we are throwing exception if clusterConfig is null. Probably we want to keep the same behavior..?

Reply:
Thanks. The method is called from other places too, so clusterConfig could be null. Let me drop this change, it's not essential anyway.